src/app/App.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import styles from './styles';
+import Library from './screens/Library';
+
+declare const global: {HermesInternal: null | {}};
+
+const App = () => {
+  return (
+    <Library
```

Comment:
i would change the name to be more like it as what it is in the app, a book item

Reply:
Done